core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -124,61 +124,82 @@ export class ContributorAdminStatsTable implements OnInit {
     return num === 1 ? word : word + 's';
   }
 
-  getCountDisplay(
+  getEditTextForAttribute(
+      editCount: number,
+      contributionSubType: string): string {
+    return ' (' + editCount + ' ' +
+           (
+           contributionSubType ===
+           AppConstants.CONTRIBUTION_STATS_SUBTYPE_REVIEW ?
+           'edited' : 'without edits'
+           ) +
+           ')';
+  }
+
+  getTextForAttribute(
```

Comment:
Again I don't know what you mean by "attribute" here, since it's not one of the arguments (so I don't know what attribute you're talking about here and what it's an attribute of). Could you please clarify?

Reply:
Done